client/state/automated-transfer/upload-data/reducer.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	AUTOMATED_TRANSFER_INITIATE as INITIATE,
+	AUTOMATED_TRANSFER_UPLOAD_UPDATE as UPDATE,
+} from 'state/action-types';
+
+/**
+ * How many bytes of the currently uploading file have already been received
```

Comment:
it is! though I'm not exactly working on this part yet; not sure if anyone else is yet either